tools/kafkatool/dump.go

```diff
@@ -36,16 +37,21 @@ func (c *DumpCommand) Register(app *kingpin.Application, getKafkaClient func() *
 	c.printer = printer
 
 	cmd := app.Command("dump", "Dump Kafka topic contents")
-	cmd.Flag("topic", "Kafka topic to dump").Required().StringVar(&c.topic)
-	cmd.Flag("partition", "Kafka partition to dump or import into").Required().IntVar(&c.partition)
 	cmd.Flag("skip-first", "Skip until input record with offset N").Default("0").IntVar(&c.skipFirst)
-	cmd.Flag("offset", "Offset to start exporting from").Default("0").Int64Var(&c.exportOffsetStart)
-	cmd.Flag("export-max-records", "Maximum number of records to export").Default("1000000").IntVar(&c.exportMaxRecords)
 	cmd.Flag("file", "File to read from or write to.").Required().OpenFileVar(&c.inOutFile, os.O_RDWR|os.O_CREATE, 0600)
 
-	cmd.Command("import", "Import records from a file into a Kafka topic").Action(c.doImport)
-	cmd.Command("export", "Export records from a Kafka topic into a file").Action(c.doExport)
-	cmd.Command("print", "Print the write requests inside records dumped using this tool").Action(c.doPrint)
+	importCmd := cmd.Command("import", "Import records from a file into a Kafka topic").Action(c.doImport)
+	importCmd.Flag("topic", "Kafka topic to import into.").Required().StringVar(&c.topic)
+	importCmd.Flag("partition", "Kafka partition to import into").Required().IntVar(&c.partition)
+
+	exportCmd := cmd.Command("export", "Export records from a Kafka topic into a file").Action(c.doExport)
```

Comment:
I understand this wasn't introduced by your PR, but it's unclear to have both `dump` and `export`...

Reply:
So the final CLI experience becomes:

```shell
./kafkatool dump export <args>
./kafkatool dump print <args>
```